scripts/docs.ts

```diff
@@ -347,7 +347,9 @@ function generateMarkdownForComment(comment: SimplifiedComment): string {
   // Example section (if available)
   if (comment.example) {
     let example = resolveLinkTags(comment.example);
-    markdown += `\`\`\`tsx\n${example}\n\`\`\`\n\n`;
+    markdown += example.includes("```")
+      ? `${example}\n\n`
+      : `\`\`\`tsx\n${example}\n\`\`\`\n\n`;
```

Comment:
Necessary to get the note out of `useRevalidator`'s example code block

Reply:
We can lift the note above the example so it shows in the summary and then we shouldn't need to try to detect this - I updated that file in f96433f37